.github/workflows/inspektor-gadget.yml

```diff
@@ -828,6 +834,12 @@ jobs:
         file: inspektor-gadget-${{ github.ref_name }}.yaml
         github-token: ${{ secrets.GITHUB_TOKEN }}
         release-url: ${{ steps.create_release.outputs.upload_url }}
+    - name: Upload checksums file
+      uses: csexton/release-asset-action@v2
+      with:
+        file: inspektor-gadget-${{ github.ref_name }}_checksums.txt
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        release-url: ${{ steps.create_release.outputs.upload_url }}
```

Comment:
What about printing the checksum of the checksum file so that we add it to the release notes? For instance, [nerdctl](https://github.com/containerd/nerdctl/blob/241ee42035ace85f955a128c8450de2a59e1456c/.github/workflows/release.yml#L38) does it.

Reply:
I thought people could have a way to verify the integrity of the checksum file they get, but yeah, they should do it manually, which is not that great advantage. Let's go ahead as it is. Thanks!